### PR TITLE
feat: Add enterprise app sample

### PR DESCRIPTION
**Pull request by Akash-Nayak**
Signed-off-by: Akash Nayak <akash19nayak@gmail.com>

### CODE REVIEW

File: samples/enterprise-app/customers-tomcat-k8s/src/main/resources/persistence.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
+ jdbc.driverClassName=oracle.jdbc.driver.OracleDriver
+ jdbc.url=jdbc:oracle:thin:@10.19.2.93:1521:xe
```
Comment (ashokponkumar) [inline samples/enterprise-app/customers-tomcat-k8s/src/main/resources/persistence.properties:2]: Can we change this ip to say a dummy hostname or something like that?

File: samples/enterprise-app/gateway/src/test/resources/application-test.properties [hunk 2]
```diff
@@ -0,0 +1,19 @@
+services.orders.url=http://orders-rhoar.192.168.42.215.nip.io/orders
+services.inventory.url=http://inventory-rhoar.192.168.42.215.nip.io/products
+services.customers.url=http://customers-rhoar.192.168.42.215.nip.io/customers
```
Comment (ashokponkumar) [inline samples/enterprise-app/gateway/src/test/resources/application-test.properties:3]: Where are we getting these ips from?

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: samples/enterprise-app/customers-tomcat-k8s/src/main/resources/persistence.properties (hunk 1 above)
Comment (Akash-Nayak) [reply samples/enterprise-app/customers-tomcat-k8s/src/main/resources/persistence.properties]: Yes sure

File: samples/enterprise-app/gateway/src/test/resources/application-test.properties (hunk 2 above)
Comment (Akash-Nayak) [reply samples/enterprise-app/gateway/src/test/resources/application-test.properties]: From here- https://github.com/konveyor/mig-demo-apps/blob/7259959b620223494c84ac512760502be43b34af/apps/e2e-demo/gateway/src/main/resources/application-local.properties#L1

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**